components/sysutils/augeas/Makefile

```diff
@@ -11,26 +11,28 @@
 
 #
 # Copyright 2016 Jim Klimov
+# Copyright (c) 2016, 2017, Oracle and/or its affiliates. All rights reserved.
+# Copyright 2017 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved.
 #
 
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=	augeas
-COMPONENT_VERSION=	1.4.0
-COMPONENT_REVISION=	1
-COMPONENT_SUMMARY=	Augeas-Tools is a configuration-file editing C library and shell toolkit
+COMPONENT_VERSION= 1.9.0
+COMPONENT_SUMMARY= Augeas is a utility for editing configuration files	
 COMPONENT_SRC=	$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
-COMPONENT_ARCHIVE_HASH=	\
-  sha256:659fae7ac229029e60a869a3b88c616cfd51cf2fba286cdfe3af3a052cb35b30
-COMPONENT_ARCHIVE_URL=	\
-  http://download.augeas.net/$(COMPONENT_ARCHIVE)
+COMPONENT_ARCHIVE_HASH=	sha256:2b463d398cabc9b42747aa61d3e83ed6a93ce03d9074cf8e7a7bd3107a668343
+COMPONENT_ARCHIVE_URL= http://download.augeas.net/$(COMPONENT_ARCHIVE)
 COMPONENT_PROJECT_URL=	http://augeas.net/index.html
 COMPONENT_FMRI=	library/$(COMPONENT_NAME)
 COMPONENT_CLASSIFICATION=	System/Libraries
 COMPONENT_LICENSE=	LGPLv2.1
 COMPONENT_LICENSE_FILE=	COPYING
 
+PKG_MACROS += LIBAUGEAS_VER=so.0.23.0
```

Comment:
Where do you use this macros?

Reply:
I don't use the definitions. I guess they will be used by the auto* facility.
Comment out or remove?

Reply:
remove